tests/e2e/test_func.py

```diff
@@ -439,21 +439,51 @@ def f(x: UInt32, y: UInt32) -> UInt32:
             return inline_f(x, y)
 
 
-# In the future, we should make this NameError, but for now, this is correct
-def test_inline_func_scope():
+def test_inline_func_scope_good():
+    c = 56
```

Comment:
Technically this variable `c` is in scope for both the inline function and f, so it doesn't verify that it uses variables outside the scope of the caller